pkg/cmd/roachtest/tests/failover.go

```diff
@@ -283,10 +283,11 @@ func runFailoverChaos(ctx context.Context, t test.Test, c cluster.Cluster, readO
 		if readOnly {
 			readPercent = 100
 		}
+
 		err := c.RunE(ctx, option.WithNodes(c.WorkloadNode()), fmt.Sprintf(
 			`./cockroach workload run kv --read-percent %d --write-seq R%d `+
 				`--concurrency 256 --max-rate 2048 --timeout 1m --tolerate-errors `+
-				`--histograms=`+t.PerfArtifactsDir()+`/stats.json {pgurl:1-2}`,
+				roachtestutil.GetWorkloadHistogramArgsString(t, c, getKVLabels(256, insertCount, readPercent))+` {pgurl:1-2}`,
```

Comment:
I prefer `fmt.Sprintf` to be used throughout instead of `+`. It will be better that we do that everywhere IMO. IN this place definitely we should not use + as we are already using fmt.Sprintf.

Reply:
`+` is being used to make the code multi-line, else this piece of code can overflow over 300 characters

Reply:
I think I understand what you mean, I have updated the code. Thanks for pointing out.